lesson_7/lesson7_decorators.py

```diff
@@ -0,0 +1,65 @@
+# "GREETINGS WORLD!"
+
+def uppercase(fnc):
+    def inner_fnc(*args):
+        initial_str = fnc(*args)
+        uppercase_str = initial_str.upper()
+        return uppercase_str
```

Comment:
As a shorter variant i would return upper function in a single line:
`return fnc(*args).upper()`

Reply:
I thought explicit would be better than implicit.. :) but I have now changed the code into a single line